index.js

```diff
@@ -439,10 +440,10 @@ module.exports.annotations = [
               var sassData = exampleItem.code;
               exampleItem.rendered = undefined;
               try {
-                if (env.sassincludes) {
-                  for (var i = env.sassincludes.length - 1; i >= 0; i = i - 1) {
-                    sassData =
-                      "@import '" + env.sassincludes[i] + "';\n" + sassData;
+                if (env.herman.sassincludes) {
+                  var arr = env.herman.sassincludes;
+                  for (var i = arr.length - 1; i >= 0; i = i - 1) {
+                    sassData = "@import '" + arr[i] + "';\n" + sassData;
```

Comment:
Hm, I guess so? Our linter currently balks at any ES6 to be safe, even though _some_ ES6 features are supported by Node v4.

The other thing here is that I want to iterate over the array in reverse, but I don't want to `.reverse()` the array every time. So if switching to a `for...of` loop, I'd need to first `.reverse()` the array just once, and then iterate for every `item`.